src/pages/ProhibitorySigns/components/MapLayers/WideRoads.test.tsx

```diff
@@ -7,7 +7,7 @@ import { getGeneratedPath } from 'shared/utils/path'
 import { withApp } from '../../../../../test/utils/withApp'
 
 describe('ProhibitorySignsWideRoads', () => {
-  jest.setTimeout(15000)
+  jest.setTimeout(60000)
```

Comment:
(multiple files) These timeouts seem excessive. Is this necessary or can we change something about the tests to increase speed/reliability?

Reply:
These tests are a mess. It mostly seems related to the very out-of-date Amsterdam React Maps (ARM) library and the Jest testing suite. So lots of technical debt with ARM _and_ the testing suite. An overhaul is needed for both in my opinion.

In the meanwhile I increased the timeouts because failing tests due to timeouts is very annoyoing. Note that tests will not take 60 seconds if they pass before this time.

Reply:
Wontfix